### DISCUSSION TITLE
Dialog Auto Focus

### DISCUSSION THREAD

**Discussion by QaahirStewart** [unovue/shadcn-vue#832] [General]
Is there a way to disable the auto focus from a input field for dialog component?

**Comment by sadeghbarati** [comment]
Yes check `radix-vue` Dialog documentation

https://www.radix-vue.com/components/dialog#content

Check for `DialogContent` emits section, you can use `openAutoFocus.prevent` to prevent autoFocus behaviour

```vue
<DialogContent @openAutoFocus.prevent>
   ...
</DialogContent>
```

or 

```vue
<DialogContent @openAutoFocus="(e) => e.preventDefault()">
   ...
</DialogContent>
```

**Reply by merzavko** [reply]
thanks man!
